plugins/filter/lib/voicecall-filter-plugin.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix=/usr
+exec_prefix=${prefix}
+libdir=${prefix}/$$PKGCONFIG_LIB
```

Comment:
Could also use a default for this and avoid need to pass the value externally. Also could consider should qmake be delegated to handle the creation

For reference on both details e.g. https://github.com/sailfishos/nemo-qml-plugin-dbus/blob/master/src/nemo-dbus/nemo-dbus.pro#L53

Reply:
Very nice that QMake can handle this. I've switch to use it.